resources/css/bem/beatmapset-info.less

```diff
@@ -74,24 +64,36 @@
   }
 
   &__header {
+    position: sticky;
```

Comment:
this effect is kinda weird for all but tags since they usually only have single value row